pages/inspector/connect-inspector-to-gtm.mdx

```diff
@@ -1,14 +1,74 @@
+* Observability can be added to all Google Analytics events using our server side GTM installation option
+* Events detected by Inspector can be added to the Avo tracking plan within the Inspector UI, allowing you to fill a documentation gap on legacy events in seconds
+
+
+Google Tag Manager (GTM) is a tag management system that allows you to quickly update tags and code snippets on your website, web apps, and apps. Once you deploy the Tag Manager snippet, you can configure (trigger) tags using a web-based user interface without having to alter and deploy additional code. It is common for early stage companies to use Google Tag Manager to take care of dispatching their core analytics events to reduce developer dependencies whenever new events are needed, and as the analytics maturity grow and more accuracy is needed, developers will deploy in-code data layer events, and then GTM tag owners will trigger the analytics service events off the data layer events accordingly. [Learn more about that here](https://www.loom.com/share/22c9e0f1175141efa7a0bdbf51201096). 
+
+**_Reminders:_** 
+- The Inspector SDKs are designed with flexibility in mind, so you should always be able to add observability to the source of your pipelines.
+- Avo Inspector operates on the environment level, please keep it in mind and create separate tags for development, staging and production data.
+
+## Getting Started
+Avo Inspector requires two core steps to add observability:
+1. Initializing the Inspector
+2. Sending events to Inspector
+
+Both steps have installation options available to you to support different configurations. Which ever path you take, your data observability journey starts with creating an Avo source because this is where you create source specific API keys and can get the initialization code. If you haven't created a source yet, follow the docs to [create and set up your GTM source](/inspector/configuring-inspector-sources).
+
+## Inspector Client side GTM integration (beta)
+
+There are two options to install Inspector via Google Tag Manager. You can either import our GTM recipe to add observability to all data layer events, or use our Inspector function to add observability to individual events.
+
+<Callout type="info" emoji="💡">
+We recommend starting by installing the Inspector on your development or testing GTM container and connect it to the Inspector dev environment. That is both so your Inspector production environment is not contaminated by testing data and because the Inspector development environment is designed to handle low data volumes real time while the production environment is optimized for large amount of production data and has 1 - 2 hour delay.
+</Callout>
+
+### Importable GTM recipe
+
+To add observability to all data layer events with our importable GTM recipe [in under 90 seconds](https://www.loom.com/share/c659b857d4d947729683bf11691f441e), follow the steps below:
+1. [Download our GTM recipe here]()
```

Comment:
@thoragudf if Alex signs off the recipe after my last comment in Asana, we need to host the code somewhere they can download it when clicking this link.